src/pages/detail-page/subpages/components/NavigatablePanel.tsx

```diff
@@ -13,13 +19,37 @@ const NavigatablePanel: React.FC<NavigatablePanelProps> = ({
   isLoading,
 }) => {
   const bigPosition = 99999;
+  const panelVisibleHeight = 1480;
+  const resizeDelay = 400;
```

Comment:
fixed arbitrary constants should be clearly distinct from others

```javascript
const BIG_POSITION = 99999;
const PANEL_VISIBLE_HEIGHT = 1480;
const RESIZE_DELAY = 400;
```
also move them out to the top, but below the imports. also add some comments for why those numbers but not something else, or a simple explanation.